content/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lifecycle badges
+---
+
+![](/images/lifecycle/lifecycle.png)
+
+### Experimental ![](https://img.shields.io/badge/lifecycle-experimental-orange.svg) {#experimental}
+
+Very early days, a lot of churn in search of a good API.  Not on CRAN. Might not go anywhere. Use with care.
+
+### Maturing ![](https://img.shields.io/badge/lifecycle-maturing-blue.svg) {#maturing}
+
+API roughed out, but finer details likely to change. Will strive to maintain backward compatibility, but need wider usage in order to get more feedback.
+
+### Stable ![](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg) {#stable}
+
+We're happy with the API, and the unlikely to be major changes. Backward incompatible changes will only be made if absolutely critical, and will be accompanied by a change in the major version.
```

Comment:
In a stable package, we are largely happy with the API, and major changes are unlikely. This means that the API will generally evolve by adding new functions and new arguments; we will avoid removing arguments or changing the meaning of existing arguments.

If API breaking change are needed, they will occur gradually. To begin with, the function or argument will be deprecated; it will continue to work but will emit an message informing you of the change. Next, typically after at least a year, the message will be transformed to an error.